projects/Canary/Models/Record.cs

```diff
@@ -172,28 +195,51 @@ public string FhirInfo
             }
         }
 
-        public Record()
+        public Record(bool retrieveFsh = false)
```

Comment:
Not a fan of including FSH flag in the constructor. Is it not possible to handle FSH/SUSHI processing outside of the constructor?

Reply:
Because of the original code for this and using flags in other ctors, I went with that design.

Reply:
This has been removed and pushed into the CheckGet method